Dockerfile

```diff
@@ -9,9 +9,9 @@ ENV GOMODCACHE=/tmp/cache/go-mod
 ENV GOBIN=/usr/local/bin
 ENV GOFLAGS="-mod=mod"
 ENV GOLANGCI_LINT_CACHE=/tmp/cache/golangci-lint
-RUN mkdir -p $GOCACHE && chmod 777 $GOCACHE && \
-    mkdir -p $GOMODCACHE && chmod 777 $GOMODCACHE && \
-    mkdir -p $GOLANGCI_LINT_CACHE && chmod 777 $GOLANGCI_LINT_CACHE
+RUN mkdir -p $GOCACHE && chmod -R 777 $GOCACHE && \
```

Comment:
Z nejakeho dovodu mi to v aktualnej verzii image nejde bez toho (ako bezny user) -> musia sa nastavit prava aj pre pod-adresare.